integrations/helm/release/release.go

```diff
@@ -171,43 +172,33 @@ func (r *Release) canDelete(name string) (bool, error) {
 // TODO(michael): cloneDir is only relevant if installing from git;
 // either split this procedure into two varieties, or make it more
 // general and calculate the path to the chart in the caller.
-func (r *Release) Install(chartPath, releaseName string, fhr flux_v1beta1.HelmRelease, action Action, opts InstallOptions, kubeClient *kubernetes.Clientset) (*hapi_release.Release, error) {
+func (r *Release) Install(chartPath, releaseName string, fhr flux_v1beta1.HelmRelease, action Action, opts InstallOptions, kubeClient *kubernetes.Clientset) (*hapi_release.Release, string, error) {
```

Comment:
My initial thought was to just calculate the checksum based on the `*hapi_release.Release` we return here, during my tests I discovered however that it is possible to receive a `nil, err` while the release has been rolled out (but it failed and will be rolled back). We still need the checksum of this release that is going to be rolled back as otherwise our state check will fail.